docs/sources/installation/requirements.md

```diff
@@ -47,29 +47,25 @@ Grafana supports the following databases:
 - MySQL
 - PostgreSQL
 
-Per default Grafana ships with and uses SQLite which is an embedded database stored on disk in Grafana's installation location.
+Per default Grafana ships with and uses SQLite, which is an embedded database stored on disk in Grafana's installation location.
 
 ## Supported web browsers
 
-Grafana is supported in the following browsers:
+Grafana is supported in the current version of the following browsers. Older versions of these browsers might not be supported.
 
 - Chrome/Chromium
 - Firefox
 - Safari
 - Microsoft Edge
+- Internet Explorer 11 is only fully supported in Grafana versions prior v6.0.
 
-> Note 1: Older versions of above browsers may not be supported
+> Note: Running Grafana without JavaScript enabled in the browser is not supported.
 
-> Note 2: Internet Explorer 11 is only fully supported in Grafana versions prior v6.0.
-
-> Note 3: Running Grafana without JavaScript enabled in the browser is not supported
-
-### Known issues
-
-#### Problem with logging in using Safari 12
+### Known issue: Problem with logging in using Safari 12
```

Comment:
Why is this here rather than in Getting Started: Logging in to Grafana the first time?

Reply:
To clarify: Should I move it?

Reply:
I think it's more related to the section on supported platforms, rather than Getting started. As in, these are the platforms we support, but we're also seeing this issue.

Reply:
@ivanahuckova , can you clarify where this should go?

Reply:
I was browsing the docs and honestly, I don't know. Maybe we could have a separate documentation for known issues? Or we can just add Note 4 with this information. Safari13 has been already released, so we can link the issue with Safari 12 and add a note to upgrade to Safari 13 or to Grafana 6.3.3.
![image](https://user-images.githubusercontent.com/30407135/67616174-e1ce5000-f7d5-11e9-9442-304b6b92d5a4.png)

Reply:
Thanks for pointing that out, @ivanahuckova. I think the Safari issue is probably covered under Note 1, so we can remove the specific bit about Safari 12. I will also clarify that we recommend users should upgrade their browsers to the latest version.

Reply:
@oddlittlebird yes, that's true! Great point.